opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -1439,6 +1440,11 @@ File {} line {}.)", pattern, location.keyword, location.filename, location.linen
         ScheduleGrid grid(this->completed_cells);
         SimulatorUpdate sim_update;
         std::unordered_map<std::string, double> target_wellpi;
+        // Get the well production indices from the simulation up until now
+        // and use in the calculation for the overall production indices.
+        if (this->wellPIPointer) {
+            target_wellpi = *(this->wellPIPointer);
+        }
```

Comment:
Might this be a change of behavior outside of PYACTION?

Reply:
No, this should not change the behavior.. hm